milli/src/search/matches/mod.rs

```diff
@@ -246,9 +251,14 @@ impl<'t, A: AsRef<[u8]>> Matcher<'t, '_, A> {
         // matches needs to be counted in the crop len.
         let mut remaining_words = crop_size + first_match_word_position - last_match_word_position;
 
+        // create the initial state of the crop window: 2 iterators starting from the matches positions,
+        // a reverse iterator starting from the first match token position and going trew the beginning of the text,
```

Comment:
I am not sure what is meant by "trew" here, maybe "towards"?

Reply:
```suggestion
        // a reverse iterator starting from the first match token position and going towards the beginning of the text,
```